tests/periodic-test/cli-logs.ts

```diff
@@ -0,0 +1,57 @@
+import { Sandbox } from "npm:@e2b/code-interpreter";
+
+console.log('Starting sandbox logs test');
+
+let sandbox: Sandbox | null = null;
+let cliProcess: Promise<string> | null = null;
+
+try {
+    // Create sandbox
+    console.log('creating sandbox')
+    sandbox = await Sandbox.create();
+    console.log('Sandbox created with ID:', sandbox.sandboxId);
+
+    let strippedId = sandbox.sandboxId.split('-')[0]
+    console.log('strippedId:', strippedId)
+
+    const command = new Deno.Command("npx", {
+        args: ["@e2b/cli", "sandbox", "logs", "-f", strippedId],
+        stdout: "piped",
+        stderr: "piped",
+    });
```

Comment:
I think you don't have to do this, but it doesn't really matter

```suggestion
    const command = new Deno.Command("npx", {
        args: ["@e2b/cli", "sandbox", "logs", "-f", sandbox.sandboxId],
        stdout: "piped",
        stderr: "piped",
    });
```